src/azure-cli/azure/cli/command_modules/servicebus/custom.py

```diff
@@ -229,6 +229,7 @@ def cli_networkrule_createupdate(cmd, client, resource_group_name, namespace_nam
     NWRuleSetIpRules = cmd.get_models('NWRuleSetIpRules', resource_type=ResourceType.MGMT_SERVICEBUS)
     netwrokruleset = client.get_network_rule_set(resource_group_name, namespace_name)
 
+    logger.warning('This version will be depracated & latest version will release in breaking change release.')
```

Comment:
You can leverage `deprecate_info` for specific command to show warnings and redirections